CHANGES.md

```diff
@@ -6,6 +6,7 @@ This release supports [version
 
 ## Changes
 
+
```

Comment:
Similar question here about newlines.

Reply:
We may have over reviewed this PR now :-)